pydarn/io/superdarn.py

```diff
@@ -140,21 +143,22 @@ def missing_field_check(file_struct_list: List[dict],
         -------
         SuperDARNFieldMissing
         """
-        # Needed to get the intersection between the record and complete
-        # file structure types
-        complete_set = DarnUtilities.dict_list2set(file_struct_list)
+        """
+        We have to check each possible subset to see if there is a missing
+        field in the subset or if the whole subset is missing. If there
+        is a missing field then raise a SuperDARNFieldMissingError.
+        If a full subset is missing then this is technically okay because
+        RST will add fields, or do not included fields based on the commands,
+        and options the user uses to process the data.
+        Some fields are not included if the data is not "good" quality,
```

Comment:
Sometimes python will complain if you use " within a """ block, recommend changing "good" to 'good'

Reply:
Done